spec/features/registration/sign_up_as_an_organisation_spec.rb

```diff
@@ -188,4 +188,24 @@
       expect(page).to have_content("Email is already associated with an account. If you can't sign in, reset your password")
     end
   end
+
+  context 'when no organisation has been selected' do
+    let(:org_name_left_blank) { '' }
+
+    before { sign_up_for_account }
+
+    it 'will tell the user the organisation name cannot be left blank' do
+      update_user_details(organisation_name: org_name_left_blank)
+      within('div#error-summary') do
+        expect(page).to have_content("Organisation name can't be blank")
+      end
+    end
+
+    it 'will not display a non-whitelist error for a blank entry' do
```

Comment:
Not sure if this is the best way to test this. Its kind of like saying, "make sure a page doesn't show some totally irrelevant content". It might be better to say something like `it "only displays one error message"` and just check that the `"Organisation name can't be blank"` is the only error message displayed.

Does that make sense?